src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -68,6 +68,12 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
     from azure.cli.core.azlogging import CommandLoggerContext
     from azure.common import AzureException
     from azure.core.exceptions import AzureError
+    from requests.exceptions import SSLError
+    import traceback
+
+    logger.debug("azure.cli.core.util.handle_exception is called with an exception:")
+    # Print the trackback and exception message
```

Comment:
Is there any security concern to expose the whole traceback to users?

Reply:
1. The trackback contains no sensitive information, but only the code flow. The value of parameters are not displayed.
2. The trackback only appears in the `--debug` log and is not sent anywhere. If the user wants to share it, he/she can filter it first.
3. On the contrary, using telemetry would expose more security concern as the trackback is sent **over the network** if telemetry is turned on. 
4. Using telemetry would require us to identity the specific command execution from other millions of command executions.